packages/integration-tests/src/integration/create-multinode/1-host-discovery.spec.js

```diff
@@ -47,6 +48,20 @@ describe(`Assisted Installer Multinode Host discovery`, () => {
       bareMetalDiscoveryIsoModal.getCloseIsoButton().click();
     });
 
+    it('Should discriminate between full and minimal image', () => {
+      bareMetalDiscoveryPage.openAddHostsModal();
+      bareMetalDiscoveryIsoModal.getGenerateDiscoveryIso().click();
+      const minimalURL = bareMetalDiscoveryIsoModal.getIsoUrl();
+      bareMetalDiscoveryIsoModal.getEditISO().click();
+      bareMetalDiscoveryIsoModal.getImageType().check("full-iso")
+      bareMetalDiscoveryIsoModal.getGenerateDiscoveryIso().click();
+      const fullURL = bareMetalDiscoveryIsoModal.getIsoUrl();
+      console.log("print from here");
```

Comment:
Please remove the console logs. We'll soon automate this, but let's remember to remove them in the PRs.

FYI, you can actually concat many texts and values in a single `console.log`
eg.

`console.log('Min iso URL=', minimalURL, 'full iso URL=', fullURL)`

Reply:
That's a mistake, I'll change it